contracts/src/v0.8/dev/interfaces/FunctionsOracleInterface.sol

```diff
@@ -66,6 +66,12 @@ interface FunctionsOracleInterface {
     view
     returns (uint96);
 
+  /**
+   * @notice Set the fee charged by the DON that will be split between signing Node Operators for servicing the request
+   * @param feeAmount Fee in Juels (1e18) of LINK
+   */
+  function setRequiredFee(uint96 feeAmount) external;
```

Comment:
This one was mirroring the name of the getter which is currently at `getRequiredFee`.
I was thinking about giving the Oracle contract its own config. Is there a scenario where we would want to change the DON key? If so I'll throw them together.

Reply:
What do you mean by "change the DON key"?

Reply:
Did you mean 
```
FunctionsOracleConfig {
  donKey,
  donFee
}

setFunctionsOracleConfig(config)
```
?

I think it makes sense but we need to be careful to avoid confusion with the OCR OracleConfig.